run.go

```diff
@@ -173,7 +165,7 @@ var runCommand = &cli.Command{
 		&cli.StringFlag{
 			Name:  "runtime",
 			Usage: "Runtime to use for this container, e.g. \"crun\", or \"io.containerd.runsc.v1\"",
-			Value: plugin.RuntimeRuncV2,
+			Value: "io.containerd.runhcs.v1",
```

Comment:
Linux runtime shouldn't change

Reply:
yup! it was just to make it work.  This will need to be extracted so when built for Windows it is defaulted correctly.